build.sh

```diff
@@ -26,6 +26,7 @@
 #    sh build.sh  --fe --clean              clean and build Frontend and Spark Dpp application
 #    sh build.sh  --fe --be --clean         clean and build Frontend, Spark Dpp application and Backend
 #    sh build.sh  --spark-dpp               build Spark DPP application alone
+#    BUILD_TYPE = Debug ./build.sh --be     build CMAKE_TYPE IS Debug version for be (Default BUILD_TYPE is Release)
```

Comment:
```suggestion
#    BUILD_TYPE=Debug ./build.sh --be     build CMAKE_TYPE IS Debug version for be (Default BUILD_TYPE is Release)
```
There must not be space because `shell` won't treat it as you hope.